developer-extension/src/panel/facets.constants.ts

```diff
@@ -0,0 +1,70 @@
+// The facet id is either:
+//
+// * "$eventSource", in which case it reflects the event source (rum, logs...) and not an actual
+//   field
+//
+// * "$EVENT_SOURCE.EVENT_PATH", where:
+//   * EVENT_SOURCE is an event source (rum, logs, ...)
+//   * EVENT_PATH is the path of the event field.
+export type FacetId = string
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏Instead of crafting an FacetId, I'm wondering if we could use the Facet object ref in the different data structure the ID is used. This way we could have a source and path fields and avoid the regex to find them 
```js
export interface Facet {
  values?: { [value: FacetValue]: { label?: string } }
  label?: string
  parent?: string
  source: 'rum' | 'log'
  path: string
}
```
I don't how we could handle parent though :)

Reply:
I removed the `FacetId` concept, but kept the `"$eventSource"` special path so we can declare and handle all facets the same way.